### ISSUE TITLE
Organized Sprites' Folder

### ISSUE THREAD

**Issue by EpicGamer200245** [NKHook/NKHook5#18]
Be able to have folders in the "WeaponSprites", "TowerSprites", "BloonSprites" and others similar to these, so that you can organize the folders better, and to make this work when NKHook compiles your mod to move them over to the sprites folder.

**Comment by DisabledMallis** [comment]
Not exactly sure on what you intend with this... could you elaborate a little?

**Comment by EpicGamer200245** [comment]
What I mean is like lets take your mod Evolution for example, if you look into your tower sprites folder, it has a lot of things and can be difficult to sift through to find and alter one thing, so if you are able to make a folder in that folder named COBRA you could put all of the COBRA tower sprite files in there so that you could just go into the COBRA folder and it would be easier to find the correct file. And NKHook when compiling would move the files in the COBRA folder out of it so that BTD5 is able to use it.

**Comment by DisabledMallis** [comment]
Yes I agree. This is a feature I technically do plan on adding. It wouldnt be just sprite cells that get created with folders though... folders would represent animation cells in the texture's xml. As far as I know, animation and regular sprite cells are the same thing, but I'll have to experiment a little bit.

**Comment by DisabledMallis** [comment]
I just re-read what you wrote, and I think I misunderstood what you were intending. I thought you were talking about the actual sprites themselves, rather than the compund sprite json files. I still agree that it is a bit annoying, but I think a solution that works with all json files would be prefered. If you, or anyone else has ideas, let me know!

**Comment by EpicGamer200245** [comment]
Yeah, the reason why I only included those folders was because a lot of folders are already split up into each tower's name, if that's what you mean by "all json files"
